### PR TITLE
[DB-12357] Fetch row counts during migration assessment and send it to the sizer algorithm for better estimates

### PR DESCRIPTION
**Pull request by sanyamsinghal**
For approx row count in PG, `pg_stat_user_tables.n_live_tup` is more accurate than `pg_class.reltuples`
Because it is updated more frequently by the autovacuum process and reflects the current number of live rows more accurately.

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/3149/

**Review by makalaaneesh — APPROVED** [review]
LGTM

File: yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-row-counts.psql [hunk 1]
```diff
@@ -13,24 +13,24 @@ DECLARE
...
     FOREACH schema_name IN ARRAY schema_list LOOP
         -- Iterate over each table in the current schema
         FOR table_record IN 
-            SELECT table_schema, table_name
-            FROM information_schema.tables 
-            WHERE table_schema = schema_name AND table_type = 'BASE TABLE' 
+            SELECT s.relname AS table_name,
+                   n.nspname AS table_schema,
+                   COALESCE(NULLIF(s.n_live_tup, -1), 0) AS estimated_row_count
+            FROM pg_stat_user_tables s
```
Comment (makalaaneesh) [inline yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-row-counts.psql:24]: @sanyamsinghal pls verify that this is also supported by older PG versions (like 9)

Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-row-counts.psql:24]: yeah `pg_stat_user_tables` is available even till PG7
https://www.postgresql.org/docs/9.3/monitoring-stats.html#:~:text=Table%2027%2D5.-,pg_stat_all_tables,-View

**Pull request merged**